package.json

```diff
@@ -1,51 +1,53 @@
 {
-  "name": "ggb-mfd-project",
-  "version": "1.0.0",
-  "private": true,
-  "dependencies": {
-    "aos": "^3.0.0-beta.6",
-    "axios": "^0.21.1",
-    "bootstrap": "^4.6.0",
-    "flag-icon-css": "^3.5.0",
-    "flux": "^4.0.1",
-    "i18next": "^20.2.1",
-    "i18next-browser-languagedetector": "^6.1.0",
-    "i18next-http-backend": "^1.2.1",
-    "js-cookie": "^2.2.1",
-    "lodash.find": "^4.6.0",
-    "node-sass": "^4.14.1",
-    "react": "^16.14.0",
-    "react-animations": "^1.0.0",
-    "react-autocomplete": "^1.8.1",
-    "react-dom": "^16.14.0",
-    "react-ga": "^2.5.6",
-    "react-i18next": "^11.9.0",
-    "react-items-carousel": "^2.8.0",
-    "react-player": "^2.9.0",
-    "react-quill": "^1.3.3",
-    "react-router-dom": "^5.1.0",
-    "react-router-hash-link": "^2.4.3",
-    "react-scripts": "2.1.1",
-    "react-select": "^4.3.1",
-    "shards-react": "^1.0.3",
-    "styled-components": "^5.3.0",
-    "swr": "^0.5.6",
-    "web-vitals": "^1.1.2",
-    "xlsx": "^0.16.9"
-  },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
-  },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
-  "browserslist": [
-    ">0.2%",
-    "not dead",
-    "not ie <= 11",
-    "not op_mini all"
-  ]
+    "name": "ggb-mfd-project",
+    "version": "1.0.0",
+    "private": true,
+    "dependencies": {
+        "aos": "^3.0.0-beta.6",
+        "axios": "^0.21.1",
+        "bootstrap": "^4.6.0",
+        "flag-icon-css": "^3.5.0",
+        "flux": "^4.0.1",
+        "i18next": "^20.2.1",
+        "i18next-browser-languagedetector": "^6.1.0",
+        "i18next-http-backend": "^1.2.1",
+        "js-cookie": "^2.2.1",
+        "lodash.find": "^4.6.0",
+        "node-sass": "^4.14.1",
+        "react": "^16.14.0",
+        "react-animations": "^1.0.0",
+        "react-autocomplete": "^1.8.1",
+        "react-dom": "^16.14.0",
+        "react-ga": "^2.5.6",
+        "react-i18next": "^11.9.0",
+        "react-items-carousel": "^2.8.0",
+        "react-player": "^2.9.0",
+        "react-quill": "^1.3.3",
+        "react-router-dom": "^5.1.0",
+        "react-router-hash-link": "^2.4.3",
+        "react-scripts": "2.1.1",
+        "react-select": "^4.3.1",
+        "shards-react": "^1.0.3",
+        "styled-components": "^5.3.0",
+        "web-vitals": "^1.1.2",
+        "xlsx": "^0.16.9"
+    },
+    "scripts": {
+        "start": "react-scripts start",
+        "build": "react-scripts build",
+        "test": "react-scripts test --reporters=jest-html-reporters",
+        "eject": "react-scripts eject"
+    },
+    "eslintConfig": {
+        "extends": "react-app"
+    },
+    "browserslist": [
+        ">0.2%",
+        "not dead",
+        "not ie <= 11",
+        "not op_mini all"
+    ],
+    "devDependencies": {
```

Comment:
The only new change is new devDepen, the jest-html-reporters package. File change must be detecting tab spacing as well.